app/src/main/java/com/example/user/my_calculator/MainActivity.java

```diff
@@ -12,68 +12,45 @@
 
 public class MainActivity extends AppCompatActivity {
 
-    Button btnNumber0;
-    Button btnNumber1;
-    Button btnNumber2;
-    Button btnNumber3;
-    Button btnNumber4;
-    Button btnNumber5;
-    Button btnNumber6;
-    Button btnNumber7;
-    Button btnNumber8;
-    Button btnNumber9;
+    private TextView txtResult;
 
-    TextView txtResult;
+    private EditText edtInput;
 
-    EditText edtInput;
-
-    Button btnCE;
-    Button btnC;
-    Button btnDelete;
-
-    Button btnAdd;
-    Button btnSub;
-    Button btnMul;
-    Button btnDiv;
-
-    Button btnDot;
-    Button btnResult;
-
-    double val1=NaN;
-    double val2;
-    String ACTION;
+    private double val1=NaN;
```

Comment:
align code ctrl+alt+O/L

Reply:
also please think about more clean name for field. We usually don't use digits in naming